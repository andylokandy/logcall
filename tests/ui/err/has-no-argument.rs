#[logfn::logfn]
fn f() {}

fn main() {}
