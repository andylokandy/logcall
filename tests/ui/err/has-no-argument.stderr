error: invalid argument
 --> tests/ui/err/has-no-argument.rs:1:1
  |
1 | #[logcall::logcall]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `logcall::logcall` (in Nightly builds, run with -Z macro-backtrace for more info)
